src/midiEditorCommands.cpp

```diff
@@ -998,11 +998,12 @@ vector<MidiControlChange> getSelectedCCs(MediaItem_Take* take, int offset=-1) {
 		if (ccIndex == -1) {
 			break;
 		}
+		bool muted;
 		double position;
 		int chan, msg1, msg2, msg3;
-		MIDI_GetCC(take, ccIndex, nullptr, nullptr, &position, &msg1, &chan, &msg2, &msg3);
+		MIDI_GetCC(take, ccIndex, nullptr, &muted, &position, &msg1, &chan, &msg2, &msg3);
```

Comment:
Hmm. I don't understand why we aren't using MidiControlChange::get here instead of calling MIDI_GetCC directly. We do use MidiNote::get in getSelectedNotes. Still, that's something we can clean up separately.